dex/src/main/scala/com/wavesplatform/dex/api/http/routes/MatcherApiRoute.scala

```diff
@@ -54,26 +53,25 @@ import scala.util.Success
 
 @Path("/matcher")
 @Api()
-case class MatcherApiRoute(assetPairBuilder: AssetPairBuilder,
-                           matcherPublicKey: PublicKey,
-                           matcher: ActorRef,
-                           addressActor: ActorRef,
-                           storeEvent: StoreEvent,
-                           orderBook: AssetPair => Option[Either[Unit, ActorRef]],
-                           orderBookHttpInfo: OrderBookHttpInfo,
-                           getActualTickSize: AssetPair => BigDecimal,
-                           orderValidator: Order => FutureResult[Order],
-                           matcherSettings: MatcherSettings,
-                           matcherStatus: () => Matcher.Status,
-                           orderDb: OrderDB,
-                           time: Time,
-                           currentOffset: () => QueueEventWithMeta.Offset,
-                           lastOffset: () => Future[QueueEventWithMeta.Offset],
-                           matcherAccountFee: Long,
-                           apiKeyHash: Option[Array[Byte]],
-                           rateCache: RateCache,
-                           validatedAllowedOrderVersions: () => Future[Set[Byte]],
-                           getActualOrderFeeSettings: () => OrderFeeSettings)(implicit mat: Materializer)
+class MatcherApiRoute(assetPairBuilder: AssetPairBuilder,
```

Comment:
Why did case class become simple class?

Reply:
Because these classes don't need a comparison, thus we don't need generated `equals` and `hashCode`.